hydromt/cli/_utils.py

```diff
@@ -85,19 +83,15 @@ def parse_json(ctx, param, value: str) -> Dict[str, Any]:
 
 
 ### general parsing methods ##
-
-
 def parse_config(
     path: Optional[Union[Path, str]] = None, opt_cli: Optional[Dict] = None
 ) -> Dict:
-    """Parse config from ini `path` and combine with command line options `opt_cli`."""
+    """Parse config from yaml `path` and combine with command line options `opt_cli`."""
     opt = {}
     if path is not None and isfile(path):
         if str(path).endswith(".ini"):
-            warn(
-                "Support for .ini configuration files will be deprecated",
-                PendingDeprecationWarning,
-                stacklevel=2,
+            raise ValueError(
+                "Support for ini config files has been removed. Please convert to yaml."
             )
```

Comment:
Consider removing this. It's the same as a deprecation warning. Or consider throwing an error on everything that's not yaml.

Reply:
good point